### PR TITLE
Game creation form

### PR DESCRIPTION
**Pull request by ewelinalabas**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,26 +1,21 @@
...
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+const AppPure = (props) => {
+  if(props.phase === 0) {
+    return ( <GameForm /> );
```
Comment (balontomasz) [inline src/App.js]: bla bla bla

File: src/actions.js [hunk 2]
```diff
@@ -0,0 +1,6 @@
+const SET_BOARD_SIZE = 'SET_BOARD_SIZE'
+
+export const setBoardSize = (value) => ({
+  type: SET_BOARD_SIZE,
+  value
+})
```
Comment (balontomasz) [inline src/actions.js]: lack of empty line

File: src/components/GameForm.jsx [hunk 3]
```diff
@@ -0,0 +1,35 @@
+import React, { useState } from 'react';
+import { connect } from 'react-redux';
+import { setBoardSize } from '../actions';
+
+const GameFormPure = (props) => {
+  const [cardsNumber, setCardsNumber] = useState(4)
+  const validateInput = (value) => {
+    return (value >= 4 && value % 2 === 0) ? true : false
```
Comment (balontomasz) [inline src/components/GameForm.jsx]: try to use constants instead of magic numbers without meaning

File: src/App.js (hunk 1 above)
Comment (ewelinalabas) [reply src/App.js]: bla bla bla

**Pull request merged**